.env.docker.example

```
@@ -0,0 +1,7 @@
+# PostgreSQL
+POSTGRES_USER=cvsa
+POSTGRES_PASSWORD=password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)